XenModel/XenAPI-Extensions/PVS_site.cs

```diff
@@ -54,6 +55,39 @@ public PVS_cache_storage PvsCacheStorage(Host host)
                 pvsCacheStorage.site.opaque_ref == opaque_ref && pvsCacheStorage.host.opaque_ref == host.opaque_ref);
         }
 
+        public string NameWithWarning
+        {
+            get
+            {
+                if (!IsCacheConfigured())
+                {
+                    return string.Format(Messages.NEWVM_DEFAULTNAME, Name, Messages.PVS_CACHE_NOT_CONFIGURED);

```

Comment:
I think it would be better to not use NEWVM_DEFAULTNAME, as this is generally used to default a new object name and may change in the feature.